runtime-modules/proposals/codex/src/lib.rs

```diff
@@ -251,10 +251,11 @@ pub trait Config:
     /// `Veto Proposal` proposal parameters
     type VetoProposalProposalParameters: Get<ProposalParameters<Self::BlockNumber, BalanceOf<Self>>>;
 
+    // TODO: enable after Carthage
     /// `Update Nft limit` proposal parameters
-    type UpdateGlobalNftLimitProposalParameters: Get<
-        ProposalParameters<Self::BlockNumber, BalanceOf<Self>>,
-    >;
+    // type UpdateGlobalNftLimitProposalParameters: Get<
+    //     ProposalParameters<Self::BlockNumber, BalanceOf<Self>>,
+    // >;
```

Comment:
As I discussed with Mokhtar we should simply comment out Proposals system. The extra effort to add enabling / disabling logic is not worth anymore, since it is probably the last time we are disabling proposals.

Reply:
Correct me if I'm wrong @ignazio-bovo @mnaamani, but I think it would be just a simple change in `ensure_details_checks` from:
```
            ProposalDetails::UpdateGlobalNftLimit(..) => {
                // Note: No checks for this proposal for now
            }
```
To:
```
            ProposalDetails::UpdateGlobalNftLimit(..) => {
                false // TODO: Enable after Carthage
            }
```

Reply:
Tell me what do you think about this. Just by looking at the amount of code that needs to be commented-out in order to disable this single proposal, I think it's a way simpler and cleaner approach.